src/Caching/Storages/FileStorage.php

```diff
@@ -249,6 +249,7 @@ public function clean(array $conditions): void
 	{
 		$all = !empty($conditions[Cache::ALL]);
 		$collector = empty($conditions);
+		$namespaces = $conditions[Cache::NAMESPACE] ?? false;
```

Comment:
You can convert it to array immediately, for example `$namespace = (array) $conditions[Cache::NAMESPACE] ?? null;`